semaphore/integration_tests.py

```diff
@@ -80,26 +84,54 @@ def get_trip_plans(environment, from_place, to_place, trip_date, arrive_by):
     return requests.get(url, params).json()
 
 
-def compare_plans(plan1, plan2):
+# alerts come in random order, which sometimes causes plan comparison to fail
+def sort_alerts(json):
+    plan = json.get("plan")
+    for itinerary in plan.get("itineraries"):
+        for leg in itinerary.get("legs"):
+            alerts = leg.get("alerts")
+            if alerts and len(alerts) > 1:
+                leg["alerts"] = sorted(alerts, key=lambda a: a["alertDescriptionText"])
+
+
+def compare_plans(plan1, plan2, **kwargs):
     try:
         assert ("plan" in plan1) == ("plan" in plan2)
 
         if "plan" in plan1:
             assert ("itineraries" in plan1.get("plan")) == ("itineraries" in plan2.get("plan"))
 
             if "itineraries" in plan1.get("plan"):
+                sort_alerts(plan1)
+                sort_alerts(plan2)
+
                 j1 = json.dumps(plan1.get("plan").get("itineraries"), sort_keys=True)
                 j2 = json.dumps(plan2.get("plan").get("itineraries"), sort_keys=True)
 
                 assert j1 == j2
 
-        print("[PASS] Plans are identical, moving on\n")
+        print("[PASS] Plans are identical\n")
         return True
 
     except AssertionError:
         print("[FAIL] Plans are different:\n")
-        print(f"First plan: {json.dumps(plan1, sort_keys=True, indent=2)}\n\n\n")
-        print(f"Second plan: {json.dumps(plan2, sort_keys=True, indent=2)}\n\n\n")
+
+        if not kwargs.get("local_run", False):
```

Comment:
Better to flip this: `if kwargs.get("local_run", False): <remote> else: <local>`

Reply:
Done